server/message_creation.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+
+#CS544 - Computer Networks
+#Group 8
+#BGP Protocol
+#Team Members: Brian Quinn, TJ Rhodes, Ryan Mann, Marc Thomson
+
+#Module name: message_creation
+#Description: exposes functions to create various message types
+
+import struct
+import message
+
+def create_message(version, msg_type, payload=None, client_id=None):
+    '''The function is used to create a message.
+    Input:
+        Version: Protocol Version
+        Message Type: Protocol Message Type
+        Payload: Payload of message, Varies depending on Message
+            Used to derive Length
+        Clien ID: ID of client, Varies depending on Message
+    Output:
+        Message: Constructed message to send
+    '''
+    # Message Creation code here
+    # planning on using struct python module for parsing
+    return message
+
+# Server doesn't need to create NEWGAMETYPE Message
+
+# This Message Type doesn't exist Should add
+def create_valid_game_type_message(version):
+    '''The function is used to create a valid game type message.
+    This function passes through protocol version and
+    adds VALGAMETYPE message type for message creation
+
+    Input:
+        Version: Protocol Version
+    Output:
+        Message: return from create_message
+    '''
+    return create_message(version, ??)
+
+
+# Create INVGAMETYPE Message
+def create_invalid_game_type_message(version):
+    '''The function is used to create a invalid game type message.
+    This function passes through protocol version and
+    adds INVGAMETYPE message type for message creation
+
+    Input:
+        Version: Protocol Version
+    Output:
+        Message: return from create_message
+    '''
+    return create_message(version, 10)
```

Comment:
when we go to implement we should def enum out the numbers for the message types